src/action/base/setup_default_profile.rs

```diff
@@ -186,6 +156,9 @@ impl Action for SetupDefaultProfile {
 pub enum SetupDefaultProfileError {
     #[error("No root home found to place channel configuration in")]
     NoRootHome,
+
+    #[error("Failed to install packages with nix-env: {0}")]
+    NixEnv(crate::nixenv::NixEnvError),
```

Comment:
I think we want this instead:

```suggestion
    #[error("Failed to install packages with nix-env")]
    NixEnv(#[from] crate::nixenv::NixEnvError),
```

And then it'll compose in a nice way